res/database/Default/database_people.xml

```diff
@@ -3271,7 +3756,55 @@
 		<person id="SpeedMinister-person-id040" first_name="Douggy" last_name="Eves" first_name_original="Douglas" last_name_original="Adams" gender="1" birthday="1952-03-11" deathday="2001-05-11" imdb_id="nm0010930" creator="8605" created_by="speedminister" />
 		<person id="SpeedMinister-person-id041" first_name="Joe" last_name="Nedlloyd" first_name_original="John" last_name_original="Lloyd" gender="1" birthday="1951-09-30" deathday="" imdb_id="nm0516032" creator="8605" created_by="speedminister" />
 
-		<!-- from scr0llbaer - TODO - but here are already the people turned into references in the descriptions of already existing programmes -->
+		<!-- from Stukov -->
+
+		<person id="Stukov_GeorgeMiller_" first_name="Gabriel" last_name="Millson" first_name_original="George" last_name_original="Miller" imdb_id="nm0004306" gender="1" job="1" topgenre="2" birthday="1945-03-03" deathday="" />
+		<person id="Stukov_SandraEcheverría" first_name="Sandy" last_name="Evecharrié" first_name_original="Sandra" last_name_original="Echeverría" imdb_id="nm1214700" gender="2" job="98" birthday="1984-12-11" deathday="" />
+		<person id="Stukov_HaileeSteinfeld" first_name="Hayden" last_name="Stonefield" first_name_original="Hailee" last_name_original="Steinfeld" imdb_id="nm2794962" gender="2" job="106" birthday="1996-12-11" deathday="" />
+		<person id="Stukov_HonorDavisPye" first_name="Holton" last_name="Daves-Pie" first_name_original="Honor" last_name_original="Davis-Pye" imdb_id="nm10675999" gender="2" job="96" birthday="2009-12-11" deathday="" />
+		<person id="Stukov_ReyMysterio" first_name="Ray" last_name="Mistico" first_name_original="Rey" last_name_original="Mysterio" imdb_id="nm0347595" gender="1" job="4192"  birthday="1974-12-11" deathday=""/>
+		<person id="Stukov_ThomasCarter" first_name="Travis" last_name="Cartridge" first_name_original="Thomas" last_name_original="Carter" imdb_id="nm0141961" gender="1" job="69"  birthday="1953-07-17" deathday=""/>
+		<person id="Stukov_RandalKleiser" first_name="Randy" last_name="Kliesing" first_name_original="Randal" last_name_original="Kleiser" imdb_id="nm0459170" gender="1" job="1"  birthday="1946-07-20" deathday=""/>
+		<person id="Stukov_JoeDante" first_name="John" last_name="Danti" first_name_original="Joe" last_name_original="Dante" imdb_id="nm0001102" gender="1" job="1" topgenre="12"  birthday="1946-11-28" deathday=""/>
+		<person id="Stukov_BradfordDillman" first_name="Brantley" last_name="Dellmen" first_name_original="Bradford" last_name_original="Dillman" imdb_id="nm0226947" gender="1" job="34" topgenre="12" birthday="1930-04-14" deathday="2018-01-16" />
```

Comment:
remove topgenre

Reply:
I suppose you mean only in in the `insignificantpeople` entries. Done.